components/SwapWidget.tsx

```diff
@@ -201,7 +203,10 @@ const NETWORK_MAP: { [key: string]: string } = {
 const WXDAI = '0xe91d153e0b41518a2ce8dd3d7944fa863463a97d'
 const WETH = ['0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2', '0x6a023ccd1ff6f2045c3309768ead9e68f978f6e1']
 
-export const SwapWidget = ({ tokenSymbol, tokenImage, platforms }: SwapWidgetProps) => {
+const toUtmParams = (tokenId: string) =>
+  `utm_source=${CONFIG.utm.source}&utm_medium=${CONFIG.utm.medium}&utm_content=swap-widget-token__${encodeURI(tokenId)}`
```

Comment:
Can we use token address instead of token id?
Less practical when visually inspecting, but I assume it's better for analytics, as the id is coming from Coingecko, no?

Reply:
"as the id is coming from Coingecko", yes.

Good point, to get the address I think we can do `platforms[network]?.contractAddress`